### PR TITLE
extend unlimted offset for query iterator(#2418)

### PR DESCRIPTION
**Pull request by MrPresent-Han**
related: #2418
pr: https://github.com/milvus-io/pymilvus/pull/2420

### CODE REVIEW

File: pymilvus/orm/iterator.py [hunk 1]
```diff
@@ -37,7 +38,6 @@
     MILVUS_LIMIT,
     OFFSET,
     PARAMS,
-    PRINT_ITERATOR_CURSOR,
```
Comment (XuanYang-cn) [inline pymilvus/orm/iterator.py:40]: Please remove log=logging.getLogger(__name__) in L55

and  LOGGER.setLevel(logging.INFO) in L51

Comment (MrPresent-Han) [reply pymilvus/orm/iterator.py:40]: it's a separate file and I suppose I need to set up log as started?
log = logging.getLogger("pymilvus")
log.setLevel(logging.INFO)
changed at line 50~51

Comment (XuanYang-cn) [reply pymilvus/orm/iterator.py:40]: `pymilvus/orm/iterator.py` is part of the source codes, so don't need to getLogger from pymilvus, just use getLogger(__name__) is enough, and donot setLevel in source code, this'll break settings outside of PyMilvus(such as log settings in examples)

Loggers for examples needs to getLogger("pymilvus")

**Review by XuanYang-cn — APPROVED** [review]
/lgtm

**Review by XuanYang-cn — APPROVED** [review]

**Pull request merged**